### ISSUE TITLE
Missing alt text on Mastodon WebMentions

### ISSUE THREAD

**Issue by edent** [snarfed/bridgy#1451]
This comment has an image with alt text - https://mastodon.social/@Edent/110151774995722710

The Brid.gy response omits the alt text - https://brid.gy/comment/mastodon/@edent@mastodon.social/110151657347210229/110151774995722710

```html
<img class="u-photo" src="https://files.mastodon.social/media_attachments/files/110/151/771/251/554/330/original/a80cc6cfe970ec56.png" alt="" />
```

Is this a resurrection of #975 ?

**Comment by snarfed** [comment]
Hmm! Thanks for filing. This is the opposite direction of #975, and I'm not sure if we ever had alt text from Mastodon working. I'll look!

**Comment by snarfed** [comment]
Fixed. Thanks again!
